syft/workers/base.py

```diff
@@ -511,7 +524,11 @@ def execute_plan_command(self, message: tuple):
         return command(*args)
 
     def send_command(
-        self, recipient: "BaseWorker", message: tuple, return_ids: str = None
+        self,
+        recipient: "BaseWorker",
+        message: tuple,
+        return_ids: str = None,
+        return_value: bool = False,
```

Comment:
How does the sender decide whether or not to request a return value?

Reply:
@karlhigley This is a way to do a combined remote op + get in a single communication round
Classical workflow in PySyft requires two separate calls, and because in work on low interaction crypto protocols I suggested this optimization.
However, this should only be valid when .get() is valid ie it shouldn't provide extra rights
I have probably left a security issue here... But this idea is not supposed to allow people to do weird stuff!